### ISSUE TITLE
Automatic tagging of HAL records

### ISSUE THREAD

**Issue by kaplun** [inspirehep/inspire-next#640]
In the Holding pen, when a new record comes through the suggestion workflow, if it's tagged as rejected by ML, but the submitter is a known HAL cataloguer (list available on legacy), the record should be tagged as HAL Hidden (i.e. in MARCXML it should have 980__a:HALhidden).

cc: @annetteholtkamp

**Comment by michamos** [comment]
I don't think this is related to the guessers, but rather to the decision: if the submitter is a HAL cataloguer, then the "reject" option should correspond to having `HAL-hidden` in `special_collections`. Ideally, this should be reflected in the UI, by having the reject button text replaced by "HAL hidden".
